{"uid":"751427d40fa36b6b","name":"test_valid_sending_transfers","fullName":"tests.wallet_tests.test_wallet_send_transfer#test_valid_sending_transfers","historyId":"898d03df45f80a262ac0dcc9bda2eeff","time":{"start":1732272408854,"stop":1732272408854,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55e549a6431a <unknown>\n#1 0x55e54957a6e0 <unknown>\n#2 0x55e5495b1dc8 <unknown>\n#3 0x55e5495ad926 <unknown>\n#4 0x55e5495f96f6 <unknown>\n#5 0x55e5495f8d46 <unknown>\n#6 0x55e5495ed203 <unknown>\n#7 0x55e5495bbcc0 <unknown>\n#8 0x55e5495bcc9e <unknown>\n#9 0x55e549a31d0b <unknown>\n#10 0x55e549a35c92 <unknown>\n#11 0x55e549a1eb3c <unknown>\n#12 0x55e549a36807 <unknown>\n#13 0x55e549a040df <unknown>\n#14 0x55e549a53578 <unknown>\n#15 0x55e549a53740 <unknown>\n#16 0x55e549a63196 <unknown>\n#17 0x7fc4f018b609 start_thread","statusTrace":"request = <SubRequest 'driver' for <Function test_valid_sending_transfers>>\n\n    @pytest.fixture(autouse=True, scope=\"function\")\n    def driver(request):\n        options = webdriver.ChromeOptions()\n    \n        # Отключение анимаций\n        options.add_argument(\"--disable-animations\")\n    \n        prefs = {\"profile.managed_default_content_settings.images\": 2}  # 2 отключает загрузку изображений\n        options.add_experimental_option(\"prefs\", prefs)\n    \n        # Прочие настройки\n        options.add_argument(\"--no-sandbox\")\n        options.add_argument(\"--window-size=1280,720\")\n        options.add_argument('--disable-dev-shm-usage')\n        options.add_argument('--disable-gpu')\n    \n        # Путь к расширению\n        # project_path = os.path.dirname(os.path.abspath(__file__))\n        # extension_path = f\"{project_path}/extension/dist/chrome\"\n    \n        # CI\n        extension_path = \"/app/extension\"\n    \n        options.add_argument(f\"--load-extension={extension_path}\")\n    \n        # Инициализация драйвера\n>       driver = webdriver.Chrome(options=options)\n\n../conftest.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/chrome/webdriver.py:45: in __init__\n    super().__init__(\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/chromium/webdriver.py:66: in __init__\n    super().__init__(command_executor=executor, options=options)\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py:212: in __init__\n    self.start_session(capabilities)\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py:299: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fb18150dc70>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"session not created\",\"message\":\"session not created: Chrome failed to sta...wn>\\\\n#15 0x55e549a53740 \\\\u003Cunknown>\\\\n#16 0x55e549a63196 \\\\u003Cunknown>\\\\n#17 0x7fc4f018b609 start_thread\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\nE         (session not created: DevToolsActivePort file doesn't exist)\nE         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nE       Stacktrace:\nE       #0 0x55e549a6431a <unknown>\nE       #1 0x55e54957a6e0 <unknown>\nE       #2 0x55e5495b1dc8 <unknown>\nE       #3 0x55e5495ad926 <unknown>\nE       #4 0x55e5495f96f6 <unknown>\nE       #5 0x55e5495f8d46 <unknown>\nE       #6 0x55e5495ed203 <unknown>\nE       #7 0x55e5495bbcc0 <unknown>\nE       #8 0x55e5495bcc9e <unknown>\nE       #9 0x55e549a31d0b <unknown>\nE       #10 0x55e549a35c92 <unknown>\nE       #11 0x55e549a1eb3c <unknown>\nE       #12 0x55e549a36807 <unknown>\nE       #13 0x55e549a040df <unknown>\nE       #14 0x55e549a53578 <unknown>\nE       #15 0x55e549a53740 <unknown>\nE       #16 0x55e549a63196 <unknown>\nE       #17 0x7fc4f018b609 start_thread\n\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/errorhandler.py:229: SessionNotCreatedException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1732272399338,"stop":1732272399340,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1732272408854,"stop":1732272408993,"duration":139},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55e549a6431a <unknown>\n#1 0x55e54957a6e0 <unknown>\n#2 0x55e5495b1dc8 <unknown>\n#3 0x55e5495ad926 <unknown>\n#4 0x55e5495f96f6 <unknown>\n#5 0x55e5495f8d46 <unknown>\n#6 0x55e5495ed203 <unknown>\n#7 0x55e5495bbcc0 <unknown>\n#8 0x55e5495bcc9e <unknown>\n#9 0x55e549a31d0b <unknown>\n#10 0x55e549a35c92 <unknown>\n#11 0x55e549a1eb3c <unknown>\n#12 0x55e549a36807 <unknown>\n#13 0x55e549a040df <unknown>\n#14 0x55e549a53578 <unknown>\n#15 0x55e549a53740 <unknown>\n#16 0x55e549a63196 <unknown>\n#17 0x7fc4f018b609 start_thread\n\n","statusTrace":"  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/usr/workspace/nintondo/autotests/conftest.py\", line 33, in driver\n    driver = webdriver.Chrome(options=options)\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/chromium/webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py\", line 212, in __init__\n    self.start_session(capabilities)\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py\", line 299, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"feature","value":"Valid sending transfers from the wallet"},{"name":"tag","value":"@pytest.mark.usefixtures('driver')"},{"name":"parentSuite","value":"tests.wallet_tests"},{"name":"suite","value":"test_wallet_send_transfer"},{"name":"host","value":"b3b395eb8b5c"},{"name":"thread","value":"9-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.wallet_tests.test_wallet_send_transfer"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":2,"skipped":0,"passed":1,"unknown":0,"total":7},"items":[{"uid":"53fd290984f4cedb","reportUrl":"https://Panda2k30.github.io/nintondo_autotests/42//#testresult/53fd290984f4cedb","status":"failed","statusDetails":"AssertionError: Expected a success message, but it was not displayed.\nassert False\n +  where False = is_displayed()\n +    where is_displayed = <selenium.webdriver.remote.webelement.WebElement (session=\"754575e0a4e2fbf2bd2a76d08d0237df\", element=\"f.37BC0D742A1D1EA1A0E4471771E46944.d.62183B49BE8B707F90FA0A4ECBCC3A4D.e.58\")>.is_displayed","time":{"start":1732271512844,"stop":1732271517974,"duration":5130}},{"uid":"a2bf3355fb14e940","reportUrl":"https://Panda2k30.github.io/nintondo_autotests/41//#testresult/a2bf3355fb14e940","status":"failed","statusDetails":"AssertionError: Expected a success message, but it was not displayed.\nassert False\n +  where False = is_displayed()\n +    where is_displayed = <selenium.webdriver.remote.webelement.WebElement (session=\"a245f9dec6da9ed0efcf8fa426dc4d1a\", element=\"f.B6008BABD3BEBE4D40B6E8CF05771796.d.5EAD598BEDADA3B49D4E806D8A46118A.e.80\")>.is_displayed","time":{"start":1732114496834,"stop":1732114502066,"duration":5232}},{"uid":"3aff0b3d2fb15b2a","reportUrl":"https://Panda2k30.github.io/nintondo_autotests/40//#testresult/3aff0b3d2fb15b2a","status":"broken","statusDetails":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55a6c8b1831a <unknown>\n#1 0x55a6c862e6e0 <unknown>\n#2 0x55a6c8665dc8 <unknown>\n#3 0x55a6c8661926 <unknown>\n#4 0x55a6c86ad6f6 <unknown>\n#5 0x55a6c86acd46 <unknown>\n#6 0x55a6c86a1203 <unknown>\n#7 0x55a6c866fcc0 <unknown>\n#8 0x55a6c8670c9e <unknown>\n#9 0x55a6c8ae5d0b <unknown>\n#10 0x55a6c8ae9c92 <unknown>\n#11 0x55a6c8ad2b3c <unknown>\n#12 0x55a6c8aea807 <unknown>\n#13 0x55a6c8ab80df <unknown>\n#14 0x55a6c8b07578 <unknown>\n#15 0x55a6c8b07740 <unknown>\n#16 0x55a6c8b17196 <unknown>\n#17 0x7fb72d3ac609 start_thread","time":{"start":1732114021102,"stop":1732114021102,"duration":0}},{"uid":"5f5cd195f5053303","reportUrl":"https://Panda2k30.github.io/Tests/39//#testresult/5f5cd195f5053303","status":"failed","statusDetails":"AssertionError: Expected a success message, but it was not displayed.\nassert False\n +  where False = is_displayed()\n +    where is_displayed = <selenium.webdriver.remote.webelement.WebElement (session=\"3d9df1ae495f76476a218957722910fc\", element=\"f.DEC40D804AC69ED80874FB9F47E3C613.d.A87E1784B45E89DC5804AD049DE0F272.e.65\")>.is_displayed","time":{"start":1732008878426,"stop":1732008883337,"duration":4911}},{"uid":"c5cb002bb15be412","reportUrl":"https://Panda2k30.github.io/Tests/38//#testresult/c5cb002bb15be412","status":"passed","time":{"start":1732007007119,"stop":1732007012622,"duration":5503}},{"uid":"8f9882efcf5df3ed","reportUrl":"https://Panda2k30.github.io/Tests/37//#testresult/8f9882efcf5df3ed","status":"failed","statusDetails":"AssertionError: Expected a success message, but it was not displayed.\nassert False\n +  where False = is_displayed()\n +    where is_displayed = <selenium.webdriver.remote.webelement.WebElement (session=\"35dc316ce47d852b3b2792a44b89205c\", element=\"f.17ED124E95B62089E5F0EF9BF72D7FD9.d.1745489FBF7E409F5EE7181917F92DD2.e.57\")>.is_displayed","time":{"start":1732002832652,"stop":1732002838158,"duration":5506}}]},"tags":["@pytest.mark.usefixtures('driver')"]},"source":"751427d40fa36b6b.json","parameterValues":[]}